src/app/saga.ts

```diff
@@ -106,6 +109,8 @@ const TAG = 'app/saga'
 //    (which will put an app into `background` state until dialog disappears).
 const DO_NOT_LOCK_PERIOD = 30000 // 30 sec
 
+const STATSIG_REFRESH_PERIOD = 10000 // 10 sec for testing. TODO change to 1h
```

Comment:
nit: easier to parse visually:
```suggestion
const STATSIG_REFRESH_PERIOD = 10_000 // 10 sec for testing. TODO change to 1h
```